rskj-core/src/main/java/co/rsk/config/VmConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.config;
+
+/**
+ * Wraps configuration for the VM, which is usually derived from configuration files.
+ */
+public class VmConfig {
+
+    private final boolean vmTrace;
+    private final int vmTraceInitStorageLimit;
+    private final int dumpBlock;
+    private final String dumpStyle;
+
+    public VmConfig(RskSystemProperties config) {
+        this.vmTrace = config.vmTrace();
+        this.vmTraceInitStorageLimit = config.vmTraceInitStorageLimit();
+        this.dumpBlock = config.dumpBlock();
+        this.dumpStyle = config.dumpStyle();
+    }
```

Comment:
We should try to build VMconfig from current data instead of using data from RskSystemProperties, and have a vmConfig inside as a helper or maybe just coexist both depending the case